en/pe-docs/docs/insights/cost-analysis/cost-analysis.md

```diff
@@ -1 +1,190 @@
-# Cost Analysis
+# Choreo Cost Optimization
```

Comment:
https://github.com/wso2/docs-choreo-dev/wiki/Markdown-File-Naming-Conventions-and-Content-Formatting
As per above doc, shall we have the file name as choreo-cost-optimization